service/route53/internal/customizations/sanitizeurl.go

```diff
@@ -0,0 +1,57 @@
+package customizations
```

Comment:
It would be helpful to have a `doc.go` with package godoc describing this customization and why its here. Linked issues references would be awesome too if possible.